classes/requests/helpers/class-kp-order-lines.php

```diff
@@ -986,5 +1006,29 @@ private function get_order_coupons( $order_id ) {
 				$this->order_lines[] = $discount;
 			}
 		}
+
+		/**
+		 * PW WooCommerce Gift Cards.
+		 */
+		if ( ! empty( WC()->session->get( 'pw-gift-card-data' ) ) ) {
```

Comment:
Have you tested this? I know that pay for order links dont always add sessions to the customer. So please verify this.